clj-refactor.el

```diff
@@ -1058,10 +1058,15 @@ word test in it and whether the file lives under the test/ directory."
     (when (and cljr-add-ns-to-blank-clj-files
                (cljr--clojure-ish-filename-p (buffer-file-name))
                (= (point-min) (point-max)))
-      (clojure-insert-ns-form)
-      (newline 2)
-      (when (cljr--in-tests-p)
-        (cljr--add-test-declarations)))))
+      (let* ((ns (if (and (fboundp #'cider-expected-ns)
+                          (cider-connected-p))
+                     (cider-expected-ns)
```

Comment:
It would probably be more appropriate for `cider-expected-ns` itself to return `clojure-expected-ns` when not connected.